src/ssl.c

```diff
@@ -55284,6 +55327,85 @@ int wolfSSL_CTX_set_ecdh_auto(WOLFSSL_CTX* ctx, int onoff)
     (void)onoff;
     return WOLFSSL_SUCCESS;
 }
+
+/**
+ * set security level(wolfSSL doesn't suppor security level)
```

Comment:
Add space after "level", "suppor" should be "support"

Reply:
Thanks! Fixed